_episodes/02-numpy.md

```diff
@@ -546,6 +546,20 @@ which is the average inflammation per patient across all days.
 > > Creates a substring from index 1 up to (not including) the final index,
 > > effectively removing the first and last letters from 'oxygen'
 > {: .solution}
+>
+> How can we rewrite the slice for getting the last three characters of `element` does,
+> so it will work on any character string at least three characters in length.
+>
+> > ## Solution
+> > ~~~
+> > print('last three characters:', element[-3:])
+> ~~~
+> {: .language-python}
```

Comment:
There is a slight syntax error here that is causing a failed check on the Pull Request. I think we can fix it with another greater than symbol on the two above lines:
```
> > ~~~
> > {: .language-python}
```